api/migrations/0003_fake_data.py

```diff
@@ -152,29 +190,86 @@ def _make_teacher_under_18_recipe(self):
             is_validated=self.faker.pybool,
         )
 
-    def _make_fake_coordinators(self):
-        """Makes fake coordinators."""
-        self.coordinator_recipe.make(_quantity=AMOUNT_OF_COORDINATORS)
+    def __get_random_time_or_none(self) -> time | None:
+        return self.faker.random_element([None, self.faker.time_object()])
 
-    def _make_fake_students(self):
-        """Makes fake students."""
-        self.student_recipe.make(_quantity=AMOUNT_OF_STUDENTS)
+    def _make_group_recipe(self) -> Recipe:
+        group_common_recipe = self._make_group_common_recipe(
+            APP_NAME + ".Group", teachers_min_value=1, teachers_max_value=3
+        )
+        return group_common_recipe.extend(
+            availability_slot=lambda: self._get_random_amount_of_objects(
+                DayAndTimeSlot, min_length=10, max_length=20
+            ),
+            is_for_staff_only=self.faker.pybool,
+            language_and_level=lambda: self.faker.random_element(
+                LanguageAndLevel.objects.all()
+            ),
+            lesson_duration_in_minutes=lambda: self.faker.pyint(
+                min_value=30, max_value=120, step=30
+            ),
+            status=lambda: self.faker.random_element(GroupStatus.values),
+            start_date=self.faker.past_date,
```

Comment:
groups can also start in the future but I agree that this is irrelevant in a context of just displaying the data in the dashboard

Reply:
Tried to make it more simple :)